@app/db/scripts/dump-db.js

```diff
@@ -0,0 +1,31 @@
+const { spawn } = require("child_process");
+
+const inTests = process.env.IN_TESTS;
+if (!inTests) {
+  const connectionString = process.env.GM_DBURL;
+  if (!connectionString) {
+    throw new Error(
+      "This script should only be ran from inside graphile-migrate"
+    );
+  }
+
+  spawn(
+    process.env.PG_DUMP || "pg_dump",
+    [
+      "--no-sync",
+      "--schema-only",
+      "--no-owner",
+      "--exclude-schema=graphile_migrate",
+      "--exclude-schema=graphile_worker",
+      "--file=../../data/schema.sql",
+      `"${connectionString}"`,
+    ],
+    {
+      stdio: "inherit",
+      env: {
+        ...process.env,
+      },
```

Comment:
No need for this, the default is `process.env` and here we're cloning that object for no benefit.
```suggestion
```